src/common/snippets/src/pass/align_element_type.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <snippets/itt.hpp>
+
+#include "snippets/snippets_isa.hpp"
+#include "snippets/op/convert_saturation.hpp"
+#include "snippets/pass/align_element_type.hpp"
+#include "ngraph_ops/type_relaxed.hpp"
+#include "ngraph/op/util/op_types.hpp"
+
+#include <ngraph/rt_info.hpp>
+
+
+namespace {
+auto op_supports_only_exec_type = [](const std::shared_ptr<ov::Node> &n) -> bool {
+    auto is_specific_supported_op = [](const std::shared_ptr<ov::Node> &n) -> bool {
```

Comment:
1. It looks like it covers all of our supported ops except for Convert and FQ, so it might be easier to check that it's not one of them.
2. Imagine a person wants to support a new operation in Snippets. He/she is going to update tokenization for sure, but will likely forget to update this pass. This case would not be covered by tests (because you should know of it first to cover) and we might have a convert-related issue one day.
 To prevent this, let's think of an approach that yield correct default behavior (probably No 1 would work)

Reply:
Your variant sounds logically. Changed the logic like first point. Thanks!